addons/jam/CfgWeapons.hpp

```diff
@@ -49,7 +60,7 @@ class CfgWeapons {
         magazineWell[] = {"CBA_580x42_TYPE95"};
     };
     class arifle_CTARS_base_F : Rifle_Base_F {
-        magazineWell[] = {"CBA_580x42_TYPE95"};
+        magazineWell[] = {"CBA_580x42_TYPE95", "CBA_580x42_TYPE95_XL"};
```

Comment:
I didn't notice we had the separate drum magwell (why do we need it btw ?).
You should add it to the arifle_CTAR_base_F as well.

Reply:
Done, albeit I hate it when magazines are not represented visually.